public/project/css/global.css

```diff
@@ -1,82 +1,161 @@
-body, input, button, textarea, h1, h2, h3, p {
-	font-family: 'Open Sans', sans-serif !important;
+body,
+input,
+button,
+textarea,
+h1,
+h2,
+h3,
+p {
+  font-family: 'Open Sans', sans-serif !important;
+}
+/* Animation Type FadeIn */
+.fade-in {
+  opacity: 0;
+  -webkit-animation: fadeIn ease-in 1;
+  -moz-animation: fadeIn ease-in 1;
+  -o-animation: fadeIn ease-in 1;
+  animation: fadeIn ease-in 1;
+  -webkit-animation-fill-mode: forwards;
+  -moz-animation-fill-mode: forwards;
+  -o-animation-fill-mode: forwards;
+  animation-fill-mode: forwards;
+}
+
+/* Animation Duration */
+.duration-1s {
+  -webkit-animation-duration: 1s;
+  -moz-animation-duration: 1s;
+  -o-animation-duration: 1s;
+  animation-duration: 1s;
+}
+
+.dead-center {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.center-bottom {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.top-right {
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.global-btn {
+  color: #ffffff;
+  border: 2px solid #22c1ee;
+  border-radius: 35px;
+  height: 35px;
+  width: 200px;
 }
+
 a {
   text-decoration: none;
   color: #75cfea;
 }
+
 ul {
   padding: 0;
   margin: 0;
 }
+
 li {
   list-style: none;
 }
 /*z-indexex*/
+
 .z-1 {
   z-index: 1;
 }
+
 .z-2 {
   z-index: 2;
 }
+
 .z-3 {
   z-index: 3;
 }
+
 .z-4 {
   z-index: 4;
 }
+
 .z-5 {
   z-index: 5;
 }
+
 .z-6 {
   z-index: 6;
 }
+
 .z-7 {
   z-index: 7;
 }
+
 .z-8 {
   z-index: 8;
 }
+
 .z-9 {
   z-index: 9;
 }
 /*font-sizes*/
+
 h1 {
-  font-size:30px;
+  font-size: 30px;
 }
+
 p {
-  font-size:14px;
+  font-size: 14px;
 }
+
 button {
-	-webkit-appearance: none;
-	-webkit-border-radius:0;
+  -webkit-appearance: none;
+  -webkit-border-radius: 0;
 }
+
 input {
-	-webkit-appearance: none;
-	-webkit-border-radius:0;
+  -webkit-appearance: none;
+  -webkit-border-radius: 0;
 }
 /*Default background*/
+
 #default-background {
```

Comment:
I am not sure if I added this, or if you did. But can you change this to a class from an id.